### DISCUSSION TITLE
2 cars within same portainer & maps data

### DISCUSSION THREAD

**Discussion by kefrens** [flobz/psa_car_controller#350] [Q&A]
Hi,
Is it possible to use 2 cars using the same portainer container? I own a e208 and a 3008?
Or it's best to create 2 containers pointing to different ports?

I'm as well confused with maps data. It doesn't show on the map. I checked and I have the -r argument enabled. Is it because my e208 is only the style without the integrated gps (needs to have android auto or car play)? I still have the SOS/Peugeot Assist button though
![image](https://user-images.githubusercontent.com/25610681/163333162-cfa14b69-7fbf-4bf9-af3f-4b495de89c84.png)

cheers

**Comment by flobz** [comment]
The app doesn't handle very well multiple car. Someone with tho car could implement it in the future.
Which version do you use?

**Reply by kefrens** [reply]
I'm using the flobz/psa_car_controller:latest	(installed on the 08/04)

I've installed it using portainer on a VM on a freebox delta
